src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const booksSlice = createSlice({
+  name: 'book',
+  initialState: [],
+  reducers: {
+    addBook: (state, action) => {
+      state.push(action.payload);
+    },
+    removeBook: (state, action) => {
+      state.filter((book) => book.id !== action.payload.id);
```

Comment:
The `filter()` method here does not modify the existing array but returns a new filtered array. Therefore, you either need to assign the filtered array back to the state for the changes to take effect or put a `return` statement before filtering. Kindly update your code accordingly.